ditto/src/test/java/eu/xenit/testing/ditto/internal/DefaultNodeViewTest.java

```diff
@@ -0,0 +1,31 @@
+package eu.xenit.testing.ditto.internal;
+
+import static eu.xenit.testing.ditto.internal.DittoAssertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import eu.xenit.testing.ditto.api.AlfrescoDataSet;
+import eu.xenit.testing.ditto.api.NodeView;
+import eu.xenit.testing.ditto.api.data.ContentModel.Content;
+import org.junit.jupiter.api.Test;
+
+class DefaultNodeViewTest {
+
+    @Test
+    void testCompanyHome_present() {
+        NodeView nodeView = AlfrescoDataSet.bootstrapAlfresco().build().getNodeView();
+
+        assertThat(nodeView.getCompanyHome())
+                .hasValueSatisfying(companyHome -> assertThat(companyHome)
+                        .hasNodeId(13)
```

Comment:
Name not important?

Reply:
Display name might differ between languages, but primary path will always be the same